app/views/hyrax/base/_show_actions.html.erb

```diff
@@ -38,6 +38,12 @@
     <div class='edit-button-sections'>
     <%= link_to "Delete", [main_app, presenter], class: 'btn btn-danger delete-button-show-page', data: { confirm: "Delete this #{presenter.human_readable_type}?" }, method: :delete %>
     </div>
+    <% if @presenter.workflow.actions.present? && @presenter.solr_document['workflow_state_name_ssim'] && @presenter.solr_document['workflow_state_name_ssim'].first.capitalize == 'Deposited' %>
```

Comment:
You should be able to shorten this conditional with safe navigation: `&.`
`presenter.solr_document['workflow_state_name_ssim']&.first&.capitalize == 'Deposited'`
which will allow you to remove the `@presenter.solr_document['workflow_state_name_ssim']` presence check

Reply:
or remove the `#capitalize` and compare like you do in `app/views/hyrax/base/show.html.erb`